chromium_src/ui/views/controls/menu/menu_config.cc

```diff
@@ -24,10 +24,9 @@ const MenuConfig& MenuConfig::instance() {
       auto& mutable_config = const_cast<MenuConfig&>(config);
       // Each platform sets its own config in its Init().
       // Apply our config globally at once after Init() is done.
-      mutable_config.item_vertical_margin = 4;
```

Comment:
It's deleted because we'll use upstream's value always for vertical margin?

Reply:
Yes, https://github.com/brave/brave-browser/issues/39127#issuecomment-2262856146